concordium-std/src/test_infrastructure.rs

```diff
@@ -1676,6 +1710,10 @@ impl<State: Serial + DeserialWithState<TestStateApi>> TestHost<State> {
         self.query_contract_balances.borrow_mut().insert(address, balance);
     }
 
+    /// Set the current exchange rates.
+    /// This is used for resolving a query for the current exchange rates.
+    ///
+    /// Note: Setting this overwrites previous set exchange rates.
```

Comment:
```suggestion
    /// Note: Setting this overwrites the previously set exchange rates.
```

Reply:
The other setter methods panic on duplicate invocations with the same parameter. Why did you decide that this should just overwrite?

Reply:
No, none of the current ones do, so I decided to follow that.
I think it would make sense to panic on duplicate invocations with the same parameter.

Reply:
Ah, my mistake. Nvm then ;)